### DISCUSSION TITLE
Privacy manifest file / Privacy Rules

### DISCUSSION THREAD

**Discussion by gbdavid2** [pointfreeco/swift-snapshot-testing#854] [General]
Hey, 

Apple introduced [new privacy rules](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files) last year and they are enforcing them from May 1, 2024. 

Does the snapshot testing library need a privacy manifest file and any declaration, or is this something that the team has been looking at recently?

**Comment by mbrandonw** [comment]
Hi @gbdavid2, I don't think we need to include this since it's just a testing library. The code of this library should probably never be compiled into a final app target. 

Since this is not an issue with the library I am going to convert this to a discussion.
